client/devdocs/design/typography.jsx

```diff
@@ -26,7 +26,10 @@ export default class Typography extends React.PureComponent {
 					<h2>Interface Typography</h2>
 
 					<p>
-						We use system fonts which improve the page rendering speed.<br />
+						We use system fonts for UI elements, like banners, buttons, or navigation. System fonts
```

Comment:
I wonder if we should translate this text?

Reply:
As in make it available for translation for i18n, or something else?